addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -337,7 +343,9 @@ class ChatWindow extends Component {
         if (!this.chatWindow.thread) {
             return;
         }
-        this._saveThreadScrollTop();
+        if (this._threadRef.comp) {
```

Comment:
There is a preemptive return on thread just above that should be enough to cover this case too no?

Reply:
Mmmh I'll check but there is a t-if to show/hide the component but I don't think we remove the link between chat_window and thread models

Reply:
I confirm, the link is still here but the component is not in the view so we need to be sure threadRef.comp exists